articles/azure-maps/about-azure-maps.md

```diff
@@ -228,7 +228,7 @@ To access Azure Maps services, go to the [Azure portal](https://portal.azure.com
 
 Azure Maps uses a key-based authentication scheme. When you create your account, two keys are generated. To authenticate for Azure Maps services, you can use either key.
 
-Note - Azure Maps shares customer-provided address/location queries ("Queries") with third-party TomTom for mapping functionality purposes. Queries aren't linked to any customer or end user when shared with TomTom and can't be used to identify individuals. The Mobility and Weather services, which include integration with Moovit, and AccuWeather are currently in [PREVIEW](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).
+Note - Azure Maps shares customer-provided address/location queries ("Queries") with third-party TomTom for mapping functionality purposes. Queries aren't linked to any customer or end user when shared with TomTom and can't be used to identify individuals. The Mobility services, which include integration with Moovit, are currently in [PREVIEW](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).
```

Comment:
Suggestions:

- Make this a note.
- _PREVIEW_ should be lowercase: preview
- Remove the _("Queries")_.


> [!NOTE]
> Azure Maps shares customer-provided address/location queries with third-party TomTom for mapping functionality purposes. Queries aren't linked to any customer or end user when shared with TomTom and can't be used to identify individuals. The Mobility services, which include integration with Moovit, are currently in [preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).